.php_cs

```diff
@@ -0,0 +1,61 @@
+<?php
+
+if (class_exists('Symfony\CS\Config\Config')) {
+    // PHP-CS-Fixer 1.x syntax (deprecated)
```

Comment:
don't need this, we only support one version and that is 2.x. the if cases where just needed during transition from 1.x ot 2.x